### PR TITLE
Feat: Including logger in adapter.d.ts to be exposed in UniversalContext

### PR DESCRIPTION
**Pull request by maximilianvoss**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/adapter.d.ts [hunk 1]
```diff
@@ -99,6 +99,21 @@ export declare interface InvocationInfo {
   requestId?:string;
 }
 
+/**
+ * Logger
+ */
+export declare interface Logger {
+  log(args): void;
```
Comment (tripodsan) [inline src/adapter.d.ts:106]: is log really supported?

File: src/adapter.d.ts [hunk 2]
```diff
@@ -99,6 +99,21 @@ export declare interface InvocationInfo {
...
+  fatal(args): void;
```
Comment (tripodsan) [inline src/adapter.d.ts:107]: should be varargs:

```suggestion
  fatal(...args): void;
```

File: src/adapter.d.ts [hunk 3]
```diff
@@ -99,6 +99,21 @@ export declare interface InvocationInfo {
...
+  error(args): void;
+  warn(args): void;
+  info(args): void;
+  debug(args): void;
+  verbose(args): void;
+  silly(args): void;
```
Comment (tripodsan) [inline src/adapter.d.ts:113]: we dont use silly

**Review by tripodsan — CHANGES_REQUESTED** [review]

File: src/adapter.d.ts (hunk 1 above)
Comment (maximilianvoss) [reply src/adapter.d.ts:106]: Just reflecting what you have build here: https://github.com/adobe/helix-universal/blob/main/src/utils.js#L113 and there is `log` in the list

File: src/adapter.d.ts (hunk 2 above)
Comment (maximilianvoss) [reply src/adapter.d.ts]: Done

**Review by tripodsan — CHANGES_REQUESTED** [review]
varargs for all methods....

**Review by tripodsan — APPROVED** [review]

**Pull request merged**